site/config.toml

```diff
@@ -126,4 +126,12 @@ expiryDate = ["expiryDate"]
 [markup]
   [markup.goldmark]
     [markup.goldmark.renderer]
-      unsafe = true
\ No newline at end of file
+      unsafe = true
+
```

Comment:
Do we need this? We're serving static HTTP files, and from https://gohugo.io/getting-started/configuration/#configure-server, 

> This is only relevant when running hugo server

Which we aren't doing?

Reply:
It was added in #251 to fix the problem of Note fields were not showing up.

Reply:
Sorry, my bad communication.

I meant this entire PR, it's not required since it's only useful for when running `hugo server`, which we aren't doing here.

Reply:
Hmm Got it.